src/api/app/views/webui/project/_show_actions.html.haml

```diff
@@ -12,13 +12,14 @@
                                                                              open_release_requests: open_release_requests,
                                                                              release_targets: release_targets }
     = render partial: 'webui/project/show_actions/delete_project', locals: { project: project }
-  - elsif !project.is_locked?
```

Comment:
@hennevogel just to confirm, so you want to show the `request_role_addition` even the project is locked?

Reply:
okay